### DISCUSSION TITLE
Reduce a gap occupied by the logo

### DISCUSSION THREAD

**Discussion by ipoom-jeong** [greenelab/lab-website-template#277] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://ipoom-jeong.github.io/gaudi-lab/projects/

### Version of Lab Website Template you are using

1.2.2

### Description

Hello, I really appreciate your efforts and contributions in supporting the website template.

While updating my logo, I noticed that on the sub-pages, our logo takes up a lot of space, even though the original SVG file is tightly cropped, as shown below.

Could you help to reduce this gap in the header?

Thanks,

<img width="391" alt="스크린샷 2024-08-19 175614" src="https://github.com/user-attachments/assets/68a2afa3-0363-45b2-bebf-5b5f3b38fd9e">
<img width="1664" alt="스크린샷 2024-08-19 175700" src="https://github.com/user-attachments/assets/69f12e1c-e03b-4e94-b6c1-0c4c0f9efebf">

**Comment by vincerubinetti** [comment, marked as answer]
This is a CSS bug in the template, caused by your logo SVG defining a `width` and `height` attribute. Please at `width: unset;` to `.logo > *` in `/_styles/header.scss`, like so:
https://github.com/greenelab/lab-website-template/pull/274/commits/548dfebe1f077d058eed28889cbe96dc3598f9b5

Will be fixed in v1.3.0.

**Comment by ipoom-jeong** [comment]
This solution perfectly addresses the issue. Thanks!
